### PR TITLE
FEAT-4: Retrieve list of utterances from an S3 file

### PR DESCRIPTION
**Pull request by jricaldi**
Relates to #4

### CODE REVIEW

**Comment by jkelvie** [comment]
What is the structure of the S3 file?

**Comment by jricaldi** [comment]
the structure is: 

xxxxxxx
xxxxxx
xxxxx
xxxxxxxxx

for example:
[utterances.txt](https://github.com/bespoken/utterance-gatherer/files/3641018/utterances.txt)

File: server/src/lib/bucket.ts [hunk 1]
```diff
@@ -65,4 +67,46 @@ export default class Bucket {
...
+          Bucket: getConfig().BUCKET_SENTENCES_NAME,
+          Key: sentencesFilePath,
+        })
+        .promise();
+
+      const sentences = await this.sentencesFileToJSON(
+        sentenceFile.Body as Buffer[],
+        source
+      );
+      return Promise.resolve(sentences);
```
Comment (jperata) [inline server/src/lib/bucket.ts]: you can just return sentences

File: server/src/lib/bucket.ts [hunk 2]
```diff
@@ -65,4 +67,46 @@ export default class Bucket {
...
+    } catch (e) {
+      console.log('aws error', e, e.stack);
+      return Promise.reject(e);
```
Comment (jperata) [inline server/src/lib/bucket.ts]: just throw e;

File: yarn.lock [hunk 3]
```diff
@@ -8309,6 +8309,15 @@ query-string@^4.1.0:
     object-assign "^4.1.0"
     strict-uri-encode "^1.0.0"
 
+query-string@^6.8.3:
```
Comment (jperata) [inline yarn.lock]: Add a .gitattributes to avoid this file expanding like this
https://github.com/bespoken/bst/blob/master/.gitattributes

**Pull request merged**